SamplesDashboard/SamplesDashboard/ClientApp/src/views/details/Details.tsx

```diff
@@ -92,18 +92,20 @@ export default class Details extends React.Component<any, any> {
         // call the statistics function
         this.statusStatistics(this.allItems);
         this.setState({
-            items: this.allItems.sort((a, b) => (a.packageName.toLowerCase() >
-                b.packageName.toLowerCase()) ? 1 : -1),
+            items: this.sortItems(this.allItems),
             repositoryDetails: {
             name: repositoryName,
             description: data.description,
-            url: data.url
-        },
-
+                url: data.url
+            },
             isLoading: false
         });
     }
 
+    public sortItems(items: IDetailsItem[]) {
+        const sortedItems = items.sort((a, b) => (a.packageName.toLowerCase() > b.packageName.toLowerCase()) ? 1 : -1);
+        return sortedItems;
+    }
```

Comment:
It should look like this

```suggestion
    public sortItems(items: IDetailsItem[], property) {
        const sortColumn = property.toLowerCase();
        const sortedItems = items.sort((a, b) => (a[sortColumn] > b[sortColumn]) ? 1 : -1);
        return sortedItems;
    }
```
So that you can call it like so

`sortItems(allItems, packageName)`
Then put it in a separate utility file so that it can be used in other places where sorting is required

However this does not need to be fixed in this PR

Reply:
So I'll need to create a separate branch to move the code into a utility file?